index.js

```diff
@@ -21,7 +21,11 @@ const app = express();
 
 app.use(cors());
 
-app.post("/", createAppDir, upload.array("files"), async (req, res) => {
+app.get("/", (req, res) => {
```

Comment:
Great job @inno-asiimwe. Can you please provide some context on why this is important? Thanks.

Reply:
LGTM tho.